tests

```diff
@@ -294,6 +306,141 @@ EOF
 	./safe export secret/import >t/home/got 2>t/home/errors
 	yamlok
 
+	testing ${version} x509 commands
+	./safe x509 issue                                                ; exitok $? 1 # need a path
+	./safe x509 issue --signed-by secret/enoent                      ; exitok $? 1 # still need a name...
+	./safe x509 issue --signed-by secret/enoent secret/cert          ; exitok $? 1 # ca must exist
+
+	./safe x509 issue secret/x509/self-signed --name www.example.com ; exitok $? 0
+	./safe x509 validate secret/x509/self-signed                     ; exitok $? 0
+	./safe exists secret/x509/self-signed:certificate                ; exitok $? 0
+	./safe exists secret/x509/self-signed:key                        ; exitok $? 0
+	./safe exists secret/x509/self-signed:combined                   ; exitok $? 0
+	./safe exists secret/x509/self-signed:serial                     ; exitok $? 1  # not a CA
+	./safe exists secret/x509/self-signed:crl                        ; exitok $? 1  # not a CA
+	./safe x509 check secret/x509/self-signed                        ; exitok $? 0
+	./safe x509 check secret/x509/self-signed --for www.example.com  ; exitok $? 0
+	./safe x509 check secret/x509/self-signed --for mail.example.com ; exitok $? 1
+	./safe x509 check secret/x509/self-signed --for test@example.com ; exitok $? 1
+	./safe x509 check secret/x509/self-signed --for 10.40.5.6        ; exitok $? 1
+
+	./safe x509 issue secret/fail-without-name                       ; exitok $? 1
+	./safe exists secret/fail-without-name:certificate               ; exitok $? 1
+	./safe exists secret/fail-without-name:key                       ; exitok $? 1
+	./safe exists secret/fail-without-name:combined                  ; exitok $? 1
+	./safe exists secret/fail-without-name:serial                    ; exitok $? 1
+	./safe exists secret/fail-without-name:crl                       ; exitok $? 1
```

Comment:
This is a question of testing discipline.  It's technically correct as-is, since the only way there can be an existing secret is for this test to put it there.  Keep in mind that every iteration of the test suite spins up a fresh Vault _tabula rasa_, so unless we do something bone-headed like create a path named `secret/fail-without-name`, it shouldn't be an issue.